packages/jest-matchers/src/index.js

```diff
@@ -78,9 +84,27 @@ const addMatchers = (matchersObj: MatchersObject): void => {
   Object.assign(global[GLOBAL_MATCHERS_OBJECT_SYMBOL], matchersObj);
 };
 
+const _validateResult = result => {
```

Comment:
using an arrow function here also causes an unnecassary `.bind(this)` call. It'd be more efficient to use a normal function here